packages/apollo-engine-reporting/src/__tests__/durationHistogram.test.ts

```diff
@@ -0,0 +1,32 @@
+import {DurationHistogram} from "../durationHistogram";
+
+describe("Duration histogram tests", () => {
```

Comment:
I think there are a lot more DurationHistogramTest.kt tests that could be ported (eg all of testSerialize)

Reply:
Added most of the ones I felt were relevant. I left out the builder tests, and the de-serialization tests.